Makefile

```
@@ -5,6 +5,9 @@ EMACS ?= emacs
 
 SHELL := bash
 
+#Point to path of your local emacs-buttercup install
+BUTTERCUP = ../emacs-buttercup/
```

```suggestion
BUTTERCUP ?= ../emacs-buttercup/
```
this allows setting it on the command line or by environment variable.